[2022-09-25T12:24:03.122+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_dag.model.migrate_to_dbt.cast_obj scheduled__2021-01-17T00:00:00+00:00 [queued]>
[2022-09-25T12:24:03.144+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_dag.model.migrate_to_dbt.cast_obj scheduled__2021-01-17T00:00:00+00:00 [queued]>
[2022-09-25T12:24:03.145+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-09-25T12:24:03.145+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2022-09-25T12:24:03.145+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-09-25T12:24:03.190+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): model.migrate_to_dbt.cast_obj> on 2021-01-17 00:00:00+00:00
[2022-09-25T12:24:03.203+0000] {standard_task_runner.py:54} INFO - Started process 299 to run task
[2022-09-25T12:24:03.208+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dbt_dag', 'model.migrate_to_dbt.cast_obj', 'scheduled__2021-01-17T00:00:00+00:00', '--job-id', '114', '--raw', '--subdir', 'DAGS_FOLDER/dbt_dag.py', '--cfg-path', '/tmp/tmp1jwtplbw']
[2022-09-25T12:24:03.210+0000] {standard_task_runner.py:83} INFO - Job 114: Subtask model.migrate_to_dbt.cast_obj
[2022-09-25T12:24:03.212+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/dbt_dag.py
[2022-09-25T12:24:03.319+0000] {task_command.py:384} INFO - Running <TaskInstance: dbt_dag.model.migrate_to_dbt.cast_obj scheduled__2021-01-17T00:00:00+00:00 [running]> on host 2768f2fff4c2
[2022-09-25T12:24:03.509+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=isaaclucky88@gmail.com
AIRFLOW_CTX_DAG_OWNER=astronomer
AIRFLOW_CTX_DAG_ID=dbt_dag
AIRFLOW_CTX_TASK_ID=model.migrate_to_dbt.cast_obj
AIRFLOW_CTX_EXECUTION_DATE=2021-01-17T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-01-17T00:00:00+00:00
[2022-09-25T12:24:03.514+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-09-25T12:24:03.516+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '\n            cd /opt/***/migrate_to_dbt &&\n            dbt --no-write-json run --target prod --models cast_obj\n            ']
[2022-09-25T12:24:03.546+0000] {subprocess.py:86} INFO - Output:
[2022-09-25T12:24:03.556+0000] {subprocess.py:93} INFO - /bin/bash: line 3: dbt: command not found
[2022-09-25T12:24:03.557+0000] {subprocess.py:97} INFO - Command exited with return code 127
[2022-09-25T12:24:03.578+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 127.
[2022-09-25T12:24:03.586+0000] {taskinstance.py:1406} INFO - Marking task as UP_FOR_RETRY. dag_id=dbt_dag, task_id=model.migrate_to_dbt.cast_obj, execution_date=20210117T000000, start_date=20220925T122403, end_date=20220925T122403
[2022-09-25T12:24:03.610+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 114 for task model.migrate_to_dbt.cast_obj (Bash command failed. The command returned a non-zero exit code 127.; 299)
[2022-09-25T12:24:03.629+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2022-09-25T12:24:03.695+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-09-25T15:24:52.278+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_dag.model.migrate_to_dbt.cast_obj scheduled__2021-01-17T00:00:00+00:00 [queued]>
[2022-09-25T15:24:52.294+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_dag.model.migrate_to_dbt.cast_obj scheduled__2021-01-17T00:00:00+00:00 [queued]>
[2022-09-25T15:24:52.294+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-09-25T15:24:52.295+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2022-09-25T15:24:52.295+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-09-25T15:24:52.344+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): model.migrate_to_dbt.cast_obj> on 2021-01-17 00:00:00+00:00
[2022-09-25T15:24:52.351+0000] {standard_task_runner.py:54} INFO - Started process 2480 to run task
[2022-09-25T15:24:52.355+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dbt_dag', 'model.migrate_to_dbt.cast_obj', 'scheduled__2021-01-17T00:00:00+00:00', '--job-id', '106', '--raw', '--subdir', 'DAGS_FOLDER/dbt_dag.py', '--cfg-path', '/tmp/tmp6okq2nug']
[2022-09-25T15:24:52.356+0000] {standard_task_runner.py:83} INFO - Job 106: Subtask model.migrate_to_dbt.cast_obj
[2022-09-25T15:24:52.357+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/dbt_dag.py
[2022-09-25T15:24:52.484+0000] {task_command.py:384} INFO - Running <TaskInstance: dbt_dag.model.migrate_to_dbt.cast_obj scheduled__2021-01-17T00:00:00+00:00 [running]> on host 3c654011b3f0
[2022-09-25T15:24:52.652+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=isaaclucky88@gmail.com
AIRFLOW_CTX_DAG_OWNER=astronomer
AIRFLOW_CTX_DAG_ID=dbt_dag
AIRFLOW_CTX_TASK_ID=model.migrate_to_dbt.cast_obj
AIRFLOW_CTX_EXECUTION_DATE=2021-01-17T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-01-17T00:00:00+00:00
[2022-09-25T15:24:52.654+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-09-25T15:24:52.655+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '\n            cd /opt/***/migrate_to_dbt &&\n            dbt --no-write-json run --target prod --models cast_obj\n            ']
[2022-09-25T15:24:52.672+0000] {subprocess.py:86} INFO - Output:
[2022-09-25T15:24:52.675+0000] {subprocess.py:93} INFO - /bin/bash: line 3: dbt: command not found
[2022-09-25T15:24:52.676+0000] {subprocess.py:97} INFO - Command exited with return code 127
[2022-09-25T15:24:52.693+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 127.
[2022-09-25T15:24:52.698+0000] {taskinstance.py:1406} INFO - Marking task as UP_FOR_RETRY. dag_id=dbt_dag, task_id=model.migrate_to_dbt.cast_obj, execution_date=20210117T000000, start_date=20220925T152452, end_date=20220925T152452
[2022-09-25T15:24:52.717+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 106 for task model.migrate_to_dbt.cast_obj (Bash command failed. The command returned a non-zero exit code 127.; 2480)
[2022-09-25T15:24:52.768+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2022-09-25T15:24:52.914+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
